### DISCUSSION TITLE
What is the antethesis to the class (.hidden)?

### DISCUSSION THREAD

**Discussion by 96312** [tailwindlabs/tailwindcss#9404] [Help] [answered]
Reference : https://tailwindcss.com/docs/display#hidden

Just like the `invisible` class has a `visible` to help show the element at various breaking points, what is the equivalent to the `hidden` class? My goal is to remove a `div` from the page layout completely at `sm:` and have it back up on `md:`. Thank you!

**Comment by wongjn** [comment, marked as answer]
`hidden` means `display: none`, so to override this, set a different [`display` property value](https://developer.mozilla.org/en-US/docs/Web/CSS/display). This will depend on the element and what you want it to do when it is visible again. For example, you may have an element as `hidden` but then you want it to be visible and it is also actually a grid layout parent, so you would have `hidden md:grid` in this case. However, for another element, it is a `<span>` in a sentence, like `<p>Lorem ipsum <span>dolor sit</span> amet</p>` In this example, you would use `inline` to keep the element inline with the rest of the sentence with `hidden md:inline`.

For your situation, `<div>` elements have the default `display` property value of `block` so without any further context, I am going to assume this is the `display` property value you would like, in which case, you would use `md:block`.

**Reply by 96312** [reply]
Thank you for the quick answer! Thought about `md:block` or like you mentioned, what the context of the element is, had occured but I still wanted to run it through the forum.
